shacl12-core/index.html

```diff
@@ -6128,170 +6127,9 @@ <h4>sh:nodeByExpression</h4>
 						as an <a>IRI expression</a> that evaluates to a set containing the same <a>node shape</a>.
 					</p>
 					<p>
-						In the following example, all values of the property <code>ex:address</code> must fulfill the
-						constraints expressed by the <a>shape</a> <code>ex:AddressShape</code>.
+						For a simple usage example, refer to the example for <a href="#NodeConstraintComponent"><code>sh:node</code></a>,
+						but replace `sh:node` with `sh:nodeByExpression`.
```

Comment:
How many electrons do we save by this "look at the other example"? How much misunderstanding do we expect from people not quite following "replace" instructions? I think we'd be wiser to keep each example distinct, with any such "replace" already done.
```suggestion
						For a simple example of its use, refer to the example for <a href="#NodeConstraintComponent"><code>sh:node</code></a>,
						replacing `sh:node` with `sh:nodeByExpression`.
```

Reply:
> How many electrons do we save by this "look at the other example"? How much misunderstanding do we expect from people not quite following "replace" instructions? I think we'd be wiser to keep each example distinct, with any such "replace" already done.

I can always revert that change if it is decided that's best.

Reply:
It would make more sense to have an example of this in the Node Expressions spec, not in Core.

Reply:
It would be great to put an example in the Node Expressions spec or somewhere similar since an example in the core spec essentially could only be the same as `sh:node` except for the addition of `sh:sourceConstraint` in the result.